classes/class-wc-connect-api-client.php

```diff
@@ -466,7 +466,7 @@ protected function request_headers() {
 			$lang = $locale_elements[ 0 ];
 			$headers[ 'Accept-Language' ] = $locale . ',' . $lang;
 			$headers[ 'Content-Type' ] = 'application/json; charset=utf-8';
-			$headers[ 'Accept' ] = 'application/vnd.woocommerce-connect.v1';
+			$headers[ 'Accept' ] = 'application/vnd.woocommerce-connect.v2';
```

Comment:
The v2 is kinda like a magic number. I think we should put it in a constant. This is how I had it in my PoC. You should be able to rebase from https://github.com/Automattic/woocommerce-services/pull/1913/files then it'll be more obvious how to update to a newer API later.